empty-plugin/src/main/resources/archetype-resources/.gitignore

```diff
@@ -1,2 +1,19 @@
 target
+
+# mvn hpi:run
 work
+
+# mvn release:rollback release:clean
+pom.xml.releaseBackup
+release.properties
```

Comment:
matter of taste, but I personally actually don't like ignoring those two files. I strongly prefer to see them and delete them when they stay, as they are never to exist for long.

Added to .gitignore means `git status` will not show it anymore. I don't think this a perfect default behaviour.